### PR TITLE
Albion.Online version 20230705101021

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112269&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/a/Albion/Online/20230705101021/Albion.Online.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+InstallerSuccessCodes:
+- 1223
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://live.albiononline.com/clients/20230530100424/albion-online-setup.exe
+  InstallerSha256: D9BFB597A316ED9055BA7C594DCCB09035CBA332575FB9F5870872B2779703F1
+ManifestType: installer
+ManifestVersion: 1.5.0
```
Comment (CoolPlayLin) [inline manifests/a/Albion/Online/20230705101021/Albion.Online.installer.yaml:16]:
```suggestion
ManifestVersion: 1.4.0
```

File: manifests/a/Albion/Online/20230705101021/Albion.Online.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+License: Proprietary
+LicenseUrl: https://albiononline.com/en/terms_and_conditions
+Copyright: ©2012-2022 SANDBOX INTERACTIVE GMBH
+ShortDescription: Albion Online is the first truly cross-platform MMO experience, available for Windows, Mac, Linux and Android, with all players on one master server in one massive, sprawling world. You choose how your adventure evolves; at home or on the go!
+Moniker: albion
+Tags:
+- game
+- mmo
+ManifestType: defaultLocale
+ManifestVersion: 1.5.0
```
Comment (CoolPlayLin) [inline manifests/a/Albion/Online/20230705101021/Albion.Online.locale.en-US.yaml:21]:
```suggestion
ManifestVersion: 1.4.0
```

File: manifests/a/Albion/Online/20230705101021/Albion.Online.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.0.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Albion.Online
+PackageVersion: 20230705101021
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.5.0
```
Comment (CoolPlayLin) [inline manifests/a/Albion/Online/20230705101021/Albion.Online.yaml:8]:
```suggestion
ManifestVersion: 1.4.0
```

**Review by CoolPlayLin — CHANGES_REQUESTED** [review]
Automatic Validation:
> One or more errors occurred. (Manifest: The manifest version is not supported. Supported versions: 1.0.0,1.1.0,1.2.0,1.4.0.) Manifest: The manifest version is not supported. Supported versions: 1.0.0,1.1.0,1.2.0,1.4.0.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**